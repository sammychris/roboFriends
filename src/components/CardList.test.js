import { shallow } from "enzyme";
import CardList from "./CardList";

const mockRobots = [{ name: "sammy", email: "ebusam@gmail.com", id: 2 }];

const wrapper = shallow(<CardList robots={mockRobots} />);

it("expect to render cardlist", () => {
  expect(wrapper).toMatchSnapshot();
});
