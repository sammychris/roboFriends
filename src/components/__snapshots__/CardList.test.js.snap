// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cardlist 1`] = `ShallowWrapper {}`;
